### PR TITLE
deps: bump `ppom` and other deps, extract benchmarks as a module

### PR DESCRIPTION
**Pull request by genryxy**
Moved `org.cactoos` to test dependency.
Bumped `http` to `v1.0.1`., `asto` to `v1.9.0`, `vertx-server` to `0.5`. Also I had to add `org.junit.vintage` dependency because it did not work without it

### CODE REVIEW

File: benchmarks/src/main/java/com/artipie/helm/HelmAstoAddBench.java [hunk 1]
```diff
@@ -78,9 +78,12 @@ public void setup() throws IOException {
...
                     if (!name.equals("index.yaml")) {
-                        final byte[] bytes = new Unchecked<>(
-                            () -> Files.readAllBytes(file)
-                        ).value();
+                        final byte[] bytes;
+                        try {
+                            bytes = Files.readAllBytes(file);
+                        } catch (final IOException exc) {
+                            throw new ArtipieIOException(exc);
+                        }
```
Comment (olenagerasimova) [inline benchmarks/src/main/java/com/artipie/helm/HelmAstoAddBench.java:86]: hm, please, try `UncheckedIOScalar` from asto, it does exactly what is written here...

File: benchmarks/src/main/java/com/artipie/helm/HelmAstoReindexBench.java [hunk 2]
```diff
@@ -68,7 +68,12 @@ public void setup() throws IOException {
         try (Stream<Path> files = Files.list(Paths.get(HelmAstoReindexBench.BENCH_DIR))) {
             files.forEach(
                 file -> {
-                    final byte[] bytes = new Unchecked<>(() -> Files.readAllBytes(file)).value();
+                    final byte[] bytes;
+                    try {
+                        bytes = Files.readAllBytes(file);
+                    } catch (final IOException exc) {
+                        throw new ArtipieIOException(exc);
+                    }
```
Comment (olenagerasimova) [inline benchmarks/src/main/java/com/artipie/helm/HelmAstoReindexBench.java:76]: same here

File: benchmarks/src/main/java/com/artipie/helm/HelmAstoRemoveBench.java [hunk 3]
```diff
@@ -88,7 +88,12 @@ public void setup() throws IOException {
         try (Stream<Path> files = Files.list(Paths.get(HelmAstoRemoveBench.BENCH_DIR))) {
             files.forEach(
                 file -> {
-                    final byte[] bytes = new Unchecked<>(() -> Files.readAllBytes(file)).value();
+                    final byte[] bytes;
+                    try {
+                        bytes = Files.readAllBytes(file);
+                    } catch (final IOException exc) {
+                        throw new ArtipieIOException(exc);
+                    }
```
Comment (olenagerasimova) [inline benchmarks/src/main/java/com/artipie/helm/HelmAstoRemoveBench.java:96]: and here

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@genryxy thanks, check my suggestion please

**Comment by genryxy** [comment]
@olenagerasimova thanks, corrected

**Comment by genryxy** [comment]
@olenagerasimova could you please check

**Review by olenagerasimova — APPROVED** [review]
@genryxy thanks

**Comment by olenagerasimova** [comment]
@genryxy thanks, sorry for the delay

**Pull request merged**